lib/formatters/tapFormatter.mjs

```diff
@@ -37,7 +39,7 @@ export default function tapFormatter(results) {
 
 				for (const { text, severity, line, column, endLine, endColumn } of warnings) {
 					lines.push(
-						`    - message: "${text}"`,
+						`    - message: ${yaml.dump(`${text}`, { quotingType: '"', forceQuotes: true, lineWidth: -1 }).trim()}`,
```

Comment:
I only think of this now but `yml` is a super set of `json`.

I think that `JSON.stringify` might actually also work.
This has the (small) benefit of not requiring us to change our list of direct dependencies.

`json` already always quotes strings and uses double quotes. It also doesn't do line wrapping.

Reply:
Actually you also have `String.prototype.quote`.
I would know since I created `@string/prototype.inspect` package.
Let's try `JSON.stringify`.